examples/gameserver.yaml

```diff
@@ -35,7 +35,7 @@ spec:
   ports:
     # name is a descriptive name for the port
   - name: default
-    # portPolicy has two options:
+    # portPolicy has three options:
     # - "Dynamic" (default) the system allocates a free hostPort for the gameserver, for game clients to connect to
     # - "Static", user defines the hostPort that the game client will connect to. Then onus is on the user to ensure that the
     # - "Passthrough" dynamically sets the `containerPort` to the same value as the dynamically selected hostPort.
```

Comment:
Does this need an example in it, like we have in the fleet.yaml?

Reply:
Added new parameter to `example/gameserver.yaml`